generators/rest-api/templates/src/main/java/package/web/rest/ServerResource.java.ejs

```diff
@@ -19,7 +19,7 @@ public class ServerResource {
 
     @GetMapping("/services/<%= serverName %>")
     public Mono<ResponseEntity<Map<String,String>>> getServer() {
-        log.info("This is Server <%= serverName %>");
+        log.info("This is Server <%= serverName %>");z
```

Comment:
remove this alphabet "z" from the template file!
application wont be able to build with such characters in between.